### PR TITLE
Information architecture updates (in scope of Clarity)

### PR DESCRIPTION
**Pull request by agraebe**
This PR restructures the docs and removes some outdated content.

### CODE REVIEW

**Comment by jeffdomke** [comment]
This is great. 

- Think 'Core' is outdated. We decided 'Stacks Blockchain' is the name we should use at all times. 
- Assuming node setup and mining will go into 'Work with Blockstack Core'? 

Under /core the first 3 sections all relate to BNS. Maybe it could be: 

- Naming Service Guides
- Name Service Tutorials
- How to use BNS

The FAQ `/core/faq_technical.html` seems to be about all kinds of topics not just Naming Service. I think there may be some kind of structure to the FAQs where `/faqs/allfaqs` might roll up all the content. Maybe should deal with this in another PR. 

Beyond those 👍

File: README.md [hunk 1]
```diff
@@ -20,16 +20,16 @@ Blockstack is a ecosystem build around a platform. There are several types of us
...
   </tr>
   <tr>
     <th>Hub Providers</th>
-    <td>Users who sell or maintain a Gai services are hub providers.  They may also be application miners but need not be. These users may be more devops user types as opposed to developers.</td>
+    <td>Users who sell or maintain a Gai services are hub providers.  These users may be more devops user types as opposed to developers.</td>
   </tr>
   <tr>
     <th>Core service programmers</th>
-    <td>These are users that provide Blockstack CORE servers or who write Clarity contracts. These are also users who build wallets or explorers into the Blockstack platform.</td>
+    <td>These are users that run Stacks Blockchain node or who write Clarity contracts. These are also users who build wallets or explorers into the Blockorm.</td>
```
Comment (yknl) [inline README.md:32]: Looks like "Blockstack platform" got messed up here.

File: README.md [hunk 2]
```diff
@@ -295,12 +295,11 @@ You can view [the source code](https://github.com/blockstack/blockstack-core/blo
 
 ## Understand how the shared FAQs work
 
-The FAQ system servers single-sourced content that support the FAQs that appear in blockstack.org, app.co, and stacks.com site. We have FAQs that fall into these categories:
+The FAQ system servers single-sourced content that support the FAQs that appear in blockstack.org, and stacks.com site. We have FAQs that fall into these categories:
```
Comment (yknl) [inline README.md:298]: Don't think we own Stacks.com, typo from the original docs. Should probably be StacksToken.com

File: _android/tutorial.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ layout: learn
 description: Learn about the Android SDK
 permalink: /:collection/:path.html
 ---
-# Android SDK Tutorial (Pre-release)
+# Android DApps (Pre-release)
```
Comment (yknl) [inline _android/tutorial.md:6]: I think we can just remove "Pre-release".

File: _ios/tutorial.md [hunk 4]
```diff
@@ -3,7 +3,7 @@ layout: learn
 description: How to use Blockstack on iOS Mobile
 permalink: /:collection/:path.html
 ---
-# iOS SDK Tutorial (Pre-release)
+# iOS DApps (Pre-release)
```
Comment (yknl) [inline _ios/tutorial.md:6]: Can also drop the "Pre-release" here as well

File: README.md (hunk 1 above)
Comment (agraebe) [reply README.md:32]: Fixed

File: README.md (hunk 2 above)
Comment (agraebe) [reply README.md:298]: Fixed

File: _android/tutorial.md (hunk 3 above)
Comment (agraebe) [reply _android/tutorial.md:6]: Done

File: _ios/tutorial.md (hunk 4 above)
Comment (agraebe) [reply _ios/tutorial.md:6]: Done

**Pull request merged**